### PR TITLE
Add of a new attribute toggling EPEL repo installation

### PR DESCRIPTION
**Pull request by dosyfier**
### Description

Add of a new attribute `node['erlang']['install_epel_repository']` that enables toggling EPEL repository installation.
Thus, by setting this attribute to `false` (default being `true`), it is now possible to install Erlang in offline mode, by retrieving RPMs from some internal yum repository for example.

### Issues Resolved

None.

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by tas50** [comment]
@dosyfier Can you squash this all down to a single commit with the DCO signed off?

File: README.md [hunk 1]
```diff
@@ -28,7 +28,12 @@ Manages installation of Erlang via packages or source.
 
 - `node['erlang']['gui_tools']` - whether to install the GUI tools for Erlang.
 - `node['erlang']['install_method']` - Erlang installation method ("package", "source", or "esl" (for Erlang Solutions packages)).
+<<<<<<< master
+- `node['erlang']['package']['version']` - OS specific version of Erlang package to install.
+- `node['erlang']['package']['install_epel_repository']` - Boolean flag indicating whether EPEL yum repository should be installed (in which case Erlang packages may be retrieved from there).
+=======
 - `node['erlang']['package']['version']` - OS specific version of Erlang package to install
+>>>>>>> master
```
Comment (tas50) [inline README.md]: You have some merge artifacts here.

**Comment by dosyfier** [comment]
Hi @tas50 ,
first of all: sorry about the messiness, that's my first pull request ^^
I tried to merge my changes into one commit (5d88433) with DCO Signed-off-by as you asked. I had then to sync my fork with the upstream (commit f0cca41).
Unfortunely, I see that the DCO check is failed, although I commited with the "Signed-off-by" tag with `git commit -s`.
Do you know what's going wrong?
Thanks for your help

**Comment by tas50** [comment]
it looks like you merged upstream into your branch instead of just rebasing there. Git's a pile of fun in situations like this. You should be able to do a git rebae HEAD~2 here to nuke out the rebase commit.

**Comment by dosyfier** [comment]
I just did the `git rebase HEAD~2` as you advised and pushed it. Looks better for the CI :)
Is it ok for you?
I still have a warning on the pull request: "This branch is out-of-date with the base branch" with a button "Update branch". Should I do it?

**Comment by tas50** [comment]
@dosyfier something funky happened in Git land there, but I just cherry picked over the single commit you made. Sorry this turned into such a DCO ordeal...lawyers. I'll get this shipped.

**Comment by dosyfier** [comment]
No problem, I'm just sorry I messed it up at first. Next time, I'll be more careful with the DCO before creating any pull request for Chef.
Thanks again! have a good day :)

**Pull request closed without merging**